tests/e2e/specs/settings.spec.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+const { test, expect } = require( '@playwright/test' );
+
+/**
+ * Internal dependencies
+ */
+import SettingPage from '../utils/pages/settings.js';
+import ElementLocators from '../utils/element-locators.js';
+
+/**
+ * @type {import('../utils/pages/settings.js').default} settingPage
+ */
+let settingPage = null;
+
+/**
+ * @type {import('../utils/element-locators.js').default} onboardingPage
+ */
+let locator = null;
+
+/**
+ * @type {import('@playwright/test').Page} page
+ */
+let page = null;
+
+test.describe.skip( 'Reddit Settings', () => {
+	test.use( { storageState: process.env.ADMINSTATE } );
+
+	test.beforeAll( async ( { browser } ) => {
+		page = await browser.newPage();
+		settingPage = new SettingPage( page );
+		locator = new ElementLocators( page );
+
+		await settingPage.mockJetpackConnected();
+		await settingPage.mockRedditConnection( { status: 'connected' } );
+		await settingPage.mockOnboardingSetup( {
+			status: 'connected',
+			step: 'accounts',
+		} );
+	} );
+
+	test.afterAll( async () => {
+		await settingPage.closePage();
+	} );
+
+	test( 'Can see onboarding success modal', async () => {
+		await settingPage.mockJetpackConnected();
+		await settingPage.mockRedditConnection( { status: 'connected' } );
+		await page.goto(
+			'/wp-admin/admin.php?page=wc-admin&path=%2Freddit%2Fsetup',
+			{ waitUntil: 'domcontentloaded' }
+		);
+
+		await locator.getContinueToSetupButton().click();
+
+		await page.waitForURL(
+			'**/wp-admin/admin.php?page=wc-admin&path=%2Freddit%2Fsettings&onboarding=success'
+		);
+		expect( await page.url() ).toContain(
+			'/wp-admin/admin.php?page=wc-admin&path=%2Freddit%2Fsettings&onboarding=success'
+		);
+
+		await expect( locator.getOnboardingSuccessfulModal() ).toBeVisible();
+
+		await locator.getOnboardingSuccessfulCloseModalButton().first().click();
+		await expect(
+			locator.getOnboardingSuccessfulModal()
+		).not.toBeVisible();
+		await page.waitForURL(
+			'**/wp-admin/admin.php?page=wc-admin&path=%2Freddit%2Fsettings'
+		);
+		expect( await page.url() ).toContain(
+			'/wp-admin/admin.php?page=wc-admin&path=%2Freddit%2Fsettings'
+		);
+	} );
+
+	test( 'Shows all sections', async () => {
+		settingPage.goto();
+
+		await expect(
+			page.locator( '.sfw-section__header', {
```

Comment:
```suggestion
			page.locator( '.rfw-section__header', {
```

Noticed sfw at multiple places. Need to replace there as well.